src/helics/common/JsonProcessingFunctions.cpp

```diff
@@ -16,6 +16,17 @@ SPDX-License-Identifier: BSD-3-Clause
 
 namespace helics::fileops {
 
+bool looksLikeCommandLine(std::string_view testString)
+{
+    if (testString.empty()) {
+        return false;
+    }
+    if (testString.front() == '-') {
+        return true;
+    }
+    return testString.find("--") != std::string::npos;
```

Comment:
```suggestion
    return testString.front() == "--");
```

This seems like it should also be looking at just the front of the string. Or both this and the previous check need to check for whitespace preceeding these characters.

I'm guessing this functio is only used when the command line args are given in a config file, whereas running the helics apps and specifying these arguments directly should have the argument parsing handled by CLI11?

Reply:
It will only get used when the config is passed as a single string.    which could be a file of various types or command line arguments in a string that needs to be split and parsed.